IM/read_waveform.py

```diff
@@ -3,6 +3,8 @@
 
 import numpy as np
 
+from calculate_ims import EXT_IDX_DICT
```

Comment:
Can be defined here - to reduce co-dependency. And in calculate_ims EXT_IDX... and IDX_EXT... can be refactored into a component list as it isn't really used anymore.